### DISCUSSION TITLE
想參考大家時間管理的方式

### DISCUSSION THREAD

**Discussion by sRain0113** [1101-GS4719-C/Python-Community-GS4719-C#2688] [Q&A]
接近期末考了，之前學習的過程中經常發生以下狀況：無法在興趣以及學業中取得平衡，通常只要開始做某方向的事，就會不計時間代價，完全投入在裡面，最終導致另一邊失衡。
最慘的狀況是如果投入時間後沒辦法在一個月內看出成果，覺得自己根本沒有長進，以致情緒低落。
不過最近開始使用番茄鐘工作法，限制一次投入的時間，並在25分鐘內盡全力把事做好，時間到了就是得停，但可以有大約5分鐘的緩衝空間。
這樣把時間分成多塊，可以明顯感受到成就感，讓自己有動力做下一件事，並且能夠事先規劃當日時間，達成興趣與學業之間的平衡。
不過我覺得方法還可以改進，因此想在此詢問各位管理時間的方式。

**Comment by forestdew507** [comment]
過去曾經會把時間拆得很碎，希望能透過每 1-1.5 小時要做甚麼，來精準控時。不過後來發現有點累，因為實際操作後所需要的緩衝時間期時也會因為當下的情況而不同。
所以近期我的作法是只拆成上午下午晚上三個大區段，每個區段要做甚麼就先把它放進去，到時候如果早上燒腦的事情把自己榨乾了，就先用早上比較不燒腦的待辦平衡一下。當然，如果整個早上狀態都很不好，也是需要保有彈性的適時挪動一下原訂規畫。

**Comment by kzx-creat** [comment]
以前我試過把時間細切成24小時，每分每秒都有代辦事項，但結果是大失敗，這種方式超不適合我。之後改成把每天要做的是列出，再搭配番茄時鐘，成功率就高很多。我自己得出的結論是，我適合高彈性的時間管理，把自己壓太緊反而容易失敗。
每個人適合的方式都不同，多多嘗試也不錯

**Comment by DarsooK** [comment]
我認為採用番茄鐘工作法是個不錯的策略，能有效提升專注力並避免過度投入單一方向導致失衡。我覺得針對這樣的情況，可以再試著加入以下方法：

1.設定階段性目標：將長期目標分解為小目標，每週檢視成果，幫助感受到進步。
2.優先級排序：利用 Eisenhower 矩陣區分重要與緊急的事情，確保資源投入在對的地方。
3.定期反思與調整：每週花些時間檢視自己的計畫是否有效，根據需要調整時間分配。
4.適度獎勵：完成特定任務後，給自己一些小獎勵以增加持續動力。

總之，時間管理沒有萬用公式，持續試驗與調整才是關鍵。希望你能在嘗試中找到最適合自己的平衡方式！

**Comment by Elaine4488** [comment]
我會先把待辦事項列出來，並思考每一項任務需要多少時間。對於比較燒腦的事情，我會避免一次做太久，而是將它們分成幾個階段來進行；但有些任務雖然需要較長時間，但一次完成會更有效率，這時我會選擇一次做完。此外，我也會根據所在的地點來安排工作，以提高效率，讓每件事都能在最適合的環境中完成。

**Comment by yue083139** [comment]
我個人是會將所有事情分類，依照時間緊不緊急來安排先後順序，像是deadline比較早的作業我就會先做，如果都不太緊急，我會優先處理我覺得麻煩的工作，因為這類工作通常比較難以估計要花多少時間才能處理完，所以先處理才能避免出現一些意外狀況發生。

**Comment by Tchang323** [comment]
我的習慣是會把該週重要的事情列在一個便條紙上，如果比較簡單的事情，我會先處理掉，接著在完成比較困難的任務。但在時間分配的部分，我不會說哪個時段一定要完成哪些事情，因為這樣沒完成反而會造成心理上的壓力，我會盡可能先把雜事處理完，剩下時間全部去解決最困難的任務。如果同時間有兩的大任務ddl，我就會把時間大概分一半，並看自己在哪時候的精神狀態、效率比較適合處理那個任務

**Comment by Tchang323** [comment]
我覺得也要了解自己一天中的精神狀態，如果早上精神最好，那可以去做最困難的任務，效率會比較好。

**Comment by irenesunnn** [comment]
我覺得不要過度規劃的也是好的，我曾經追求非常細緻的時間規劃系統
但因為是完美主義者，反而常常因為規劃跟不上變化而感到挫折且不斷拖延執行日程
大致列出待辦的方法應該比較適用大部分的人

**Comment by 113607514** [comment]
最近我開始特別注意自己和周圍人的時間管理方式，發現每個人都有自己獨特的習慣。有些朋友總是把一天的工作細分成不同的時段，像是將每小時的任務列出，並準時完成，這樣的規劃讓他們的效率極高。而我自己則偏好稍微靈活一些的方法，會把主要任務先列出，再根據當時的情況來調整時間安排。對我來說，這樣的方式能讓我在保持彈性的同時，也能確保不會錯過重要的工作。除了時間的分配，我也很欣賞那些能在一天的工作中安排一些休息和反思時間的人。這樣不僅讓大腦有時間重啟，也讓我能夠保持長時間的專注。其實，每個人對時間的管理方式都有自己的理解，但我發現，找到最適合自己的方法，並不斷調整，才是最能幫助我高效完成事情的關鍵。

**Comment by Taoyi-Chiang** [comment]
以前會覺得一定要安排好所有的工作進度，但又缺乏自律所以最後總是變成番茄醬工作法。
後來我發現其實只要能在一定的時間內產出一定程度的成果就好，所以變得也沒有這麼要求自己，後來有些事情都是在截止前一天或兩三個小時才做。
人生工作的時間那麼長，又只活一次，實在不用這麼有壓力。

**Comment by lichilun** [comment]
關於時間管理，我會先列個清單，把最重要的事情放在最前面，然後確保每天有固定的時間做這些事。比如說早上先處理一些需要集中精神的工作，午後做些比較輕鬆的，甚至把休息的時間也規劃進去。

**Comment by Piyenparty** [comment]
我認為效率很重要，別人可以捲一整天然後晚上出去約會但我只能捲一整天，因為我都邊玩邊做事，導致自己時間管理的能力很差，所以我覺得自律是增加自己時間管理不可或缺的技能

**Comment by edawdewv** [comment]
我自己會用「優先清單＋彈性調整」的方式：每天早上或前一天晚上列出最重要的3件事，先做最關鍵的，避免被瑣事拖垮。另外也會使用番茄鐘用25分鐘集中做事，休息時完全放鬆，不看手機。如果當天沒完成，允許自己調整計劃，不苛求一定要達成全部。

**Comment by jaychang1027** [comment]
我自己會規劃每個時段要做甚麼事情，像是平日就是安排晚上是讀書時間，其他時間除了上課以外，就會去打打球，找朋友聊聊天，看看影片休閒一下，我也會把這個禮拜或是這幾天需要完成的事項列出來，然後依照進度分配一件一件完成。

**Comment by yuku-huang** [comment]
我會用番茄鐘加上週計畫表，一週抓三件最重要的事，每天早上檢查一次。有進度就記下來，不看成果，只看自己有沒有動起來，這樣比較不會焦慮也不會失衡。

**Comment by shrshu38** [comment]
我自己則是每天早上寫下當天要完成的三件最重要的事，再配合番茄鐘去執行。這樣做可以避免時間被「小事」吃掉，也能更快看到進度。

另外，有時我會用 Notion 或行事曆把興趣和讀書排進固定時段，刻意留白，幫自己設休息點。這樣心態上會更平衡，不會一直覺得在趕進度。

**Comment by brianou583** [comment]
我也有用過番茄鐘，剛開始很有效，尤其是那種「我現在只有25分鐘，就拼了」的感覺。但後來我發現如果任務本身比較需要沉浸（像讀比較難的書），我會改用「90分鐘專注 + 15分鐘放空」的方式，比較不容易被打斷靈感。

**Comment by 12388833** [comment]
我個人喜歡在早上一起床，花個10分鐘把要上的課，以及要讀的科目先列出來，這樣一整天下來，可以比較不會慌亂的行動，而是可以按表操課，然後我在任務與任務間，還會多留點空閒的時間，避免前個任務沒有完成，影響到之後的行程，當然不用列得很仔細，有個大概就好了，重點是讓自己有個概念今天要做甚麼。

**Comment by yunnnnchieh** [comment]
自己很不常寫計劃表，可能只會把要做的事寫在行事曆上，看哪些比較緊急發生，看到上面大家的方法，感覺都很值得試試看，不然常常拖到最後一刻才在極限趕工（雖然有時候這樣做會特別有效率…但會後悔一開始為什麼不這樣就好了）
也有看過有人說過90/20 原則，就是每工作 90 分鐘就休息 20 分鐘，模仿大腦的自然節奏，提高工作品質

**Comment by ichenx2** [comment]
我自己是用 iOS 內建的日曆跟提醒事項在規劃時間，會用不同顏色標籤區分課內跟課外的事，也會依照輕重緩急加上不同的標示方式，這樣一打開就知道今天該做什麼、什麼事可以晚一點處理，用內建的app的好處就是可以跨平台，我可以在手機上紀錄，然後平板跟電腦都會同步更新。番茄鐘我也有試過，我覺得它對於提升專注真的蠻有效的，但我自己更喜歡 50-10 的方式，有時候 25 分鐘對我來說有點太短，剛進入狀況就要停，50 分鐘對我來說是可以讓我好好做到某一個階段，然後休息10分鐘再繼續。

**Comment by alice20040127** [comment]
我自己感覺比較隨性，就是規劃我今天下午、晚上、可能還有半夜要做什麼，在那個區間內就做該做的事情，但我也蠻常做一做就分心去看手機什麼的，但前提就是在那個東西還沒有壓到deadline的狀態，現在的話則是因為看到旁邊的人一起床可能就開始認真在做事，就感覺有被影響到了，會一起認真的做事情。

**Comment by mattwang1222** [comment]
我以前也常陷入「全力衝一件事、其他全停擺」的狀況，後來開始學會設「時間預算」，例如：每天固定只有兩小時給興趣，強迫自己留下學業時間。搭配番茄鐘後，反而效率更高。規律讓我穩定，也降低了對成果的焦慮。

**Comment by Wei0810** [comment]
時間管理這玩意兒，說白了就是跟自己的拖延症打架XD。我是用App記錄自己一天幹掉幾顆「番茄」，看數字堆起來還挺爽的，感覺自己像在打怪升級
